SteamTogether.Scraper/Worker.cs

```diff
@@ -27,17 +27,22 @@ protected override async Task ExecuteAsync(CancellationToken stoppingToken)
             await scraper.RunSync();
         }
 
-        var schedule = NCrontab.CrontabSchedule.Parse(opts.Schedule);
+        _logger.LogInformation("Using schedule: {Schedule}", opts.Schedule);
+        var cron = CronExpression.Parse(opts.Schedule);
         while (!stoppingToken.IsCancellationRequested)
         {
-            var dateTimeService = _serviceProvider.GetRequiredService<IDateTimeService>();
-            var now = dateTimeService.GetCurrentTime();
+            var utcNow = DateTime.UtcNow;
```

Comment:
Consider using dateTimeService.GetCurrentTime() to get the current time, using this will help in the future with unittests